### PR TITLE
[MIG] sales_team_operating_unit: Migration to 17.0

### PR DESCRIPTION
**Pull request by ghost**
Depends on:

- [x] https://github.com/OCA/operating-unit/pull/667

### CODE REVIEW

File: sales_team_operating_unit/views/crm_team_view.xml [hunk 1]
```diff
@@ -22,10 +22,14 @@
         <field name="inherit_id" ref="sales_team.crm_team_view_form" />
         <field name="arch" type="xml">
             <field name="currency_id" position="before">
+                <field
+                    name="company_id"
+                    groups="!base.group_multi_company"
+                    column_invisible="True"
```
Comment (nguyenminhchien) [inline sales_team_operating_unit/views/crm_team_view.xml]: it should be `invisible="True"`

**Review by nguyenminhchien — CHANGES_REQUESTED** [review]

File: sales_team_operating_unit/views/crm_team_view.xml (hunk 1 above)
Comment (ghost) [reply sales_team_operating_unit/views/crm_team_view.xml]: Fixed. Thanks for the review 👍

**Review by nguyenminhchien — APPROVED** [review]

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM, Functional + code. Can you please rebase and remove the test.requirements?

**Review by AaronHForgeFlow — APPROVED** [review]
Thanks

**Pull request merged**